### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by abirbhattacharya82**
I have made a few changes to the existing the README.md. I hope you will like it and merge my Pull Request
This fixes issue #13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,28 @@
...
+# Quality Standards for Hacktoberfest
+##### The Following examples of pull requests that we consider to be low-quality contributions
+- Pull requests that are automated
+- Pull requests that are disruptive
+- Pull requests that are regarded by a project maintainer as a hindrance vs. helping.
+- Something that's clearly an attempt to simply +1 your pull request count for October.
+
 # Suggestions or improvements?
-Make an issue of the same and we will discuss it!
+- To Contribute make sure you have read the contributors guidlines.
```
Comment (kaustubhgupta) [inline README.md:20]: Please remove this table as I will make a better issue for this

File: contributer.md [hunk 2]
```diff
@@ -1,5 +1,40 @@
-hey 
-This is Anmol Singla
-I m from India
-I m pursuing btech in IT from Maharaja Aggersan Institute of Technology
-I m exploring a github accounts and then i founds you
+# Contributing Guidelines
```
Comment (kaustubhgupta) [inline contributer.md:1]: I suggest deleting this file. I no longer require this file

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Review by abirbhattacharya82 — COMMENTED** [review]
I have made the changes as per requirements

**Comment by kaustubhgupta** [comment]
Please make another PR as this conflict can't be resolved.

**Pull request closed without merging**